composer.lock

```diff
@@ -4,52 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
```

Comment:
Couldn't composer.lock be added to .gitignore?

Reply:
It was already added to .gitignore, this branch was probably made before that

Reply:
Gotcha